packages/wallet/backend/migrations/20230317135105_create_users.js

```diff
@@ -17,6 +17,8 @@ exports.up = function (knex) {
     table.string('rapydContactId')
     table.string('kycId')
```

Comment:
I assume we will delete the Rapyd related columns in a different PR?

Reply:
yes, deleted most of the rapyd related columns [here](https://github.com/interledger/testnet/pull/1612)
kycId one it is kept as will be replaced with GateHub info if kyc was completed